From c2add716da7803c461f77c0a6c9b1dd0894cfa6d Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Mon, 22 Apr 2013 14:38:38 +0200
Subject: Disable unsupported usb devices

Bugzilla: 903914

Patch deactivates device we do not support in RHEL7:
- usb-audio
- usb-braille
- usb-bt-dongle
- usb-net
- usb-serial
- usb-wacom-tablet
- pci-ohci

diff --git a/default-configs/pci.mak b/default-configs/pci.mak
index ee2d18d..b0b6ca3 100644
--- a/default-configs/pci.mak
+++ b/default-configs/pci.mak
@@ -2,7 +2,6 @@ CONFIG_PCI=y
 CONFIG_VIRTIO_PCI=y
 CONFIG_VIRTIO=y
 CONFIG_USB_UHCI=y
-CONFIG_USB_OHCI=y
 CONFIG_USB_EHCI=y
 CONFIG_USB_XHCI=y
 CONFIG_NE2000_PCI=y
diff --git a/default-configs/usb.mak b/default-configs/usb.mak
index 1bf9075..9a3bd09 100644
--- a/default-configs/usb.mak
+++ b/default-configs/usb.mak
@@ -2,7 +2,3 @@ CONFIG_USB_TABLET_WACOM=y
 CONFIG_USB_STORAGE_BOT=y
 CONFIG_USB_STORAGE_UAS=y
 CONFIG_USB_SMARTCARD=y
-CONFIG_USB_AUDIO=y
-CONFIG_USB_SERIAL=y
-CONFIG_USB_NETWORK=y
-CONFIG_USB_BLUETOOTH=y
